mod-source-record-manager-server/src/main/java/org/folio/verticle/JobMonitoringWatchdogVerticle.java

```diff
@@ -0,0 +1,109 @@
+package org.folio.verticle;
+
+import java.util.Date;
+import java.util.Set;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.shareddata.LocalMap;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.support.AbstractApplicationContext;
+import org.springframework.stereotype.Component;
+
+import org.folio.rest.jaxrs.model.JobExecution;
+import org.folio.rest.jaxrs.model.JobMonitoring;
+import org.folio.rest.jaxrs.model.JobProfileInfo;
+import org.folio.rest.jaxrs.model.RunBy;
+import org.folio.services.JobExecutionService;
+import org.folio.services.JobMonitoringService;
+import org.folio.spring.SpringContextUtil;
+
+@Component
+@PropertySource("classpath:application.properties")
+public class JobMonitoringWatchdogVerticle extends AbstractVerticle {
+
+  private static final Logger LOGGER = LogManager.getLogger();
+  private static AbstractApplicationContext springGlobalContext;
+
+  @Autowired
+  private JobMonitoringService jobMonitoringService;
+
+  @Autowired
+  private JobExecutionService jobExecutionService;
+
+  @Value("${job.monitoring.watchdog.timestamp}")
```

Comment:
Need better naming. 
If this is a 'job inactivity timeout', it should be reflected in parameter name

Reply:
changed